Source/Session.swift

```diff
@@ -481,6 +513,10 @@ open class Session {
     func retrier(for request: Request) -> RequestRetrier? {
         return request.interceptor ?? retrier
     }
+
+    func retryPolicies(for request: Request) -> [RetryPolicy] {
+        return request.retryPolicies + retryPolicies
```

Comment:
Total retry policies for a `Request` are the `Request` retry policies plus the `Session` policies...in the same order they were added.